test/testdrive/postgres-replication-sources.td

```diff
@@ -0,0 +1,90 @@
+# Copyright Materialize, Inc. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+$ postgres-execute connection=postgresql://postgres:password@${testdrive.postgres-host}/postgres
+CREATE TABLE numbers_${testdrive.seed} (number int PRIMARY KEY, is_prime bool, name text);
+ALTER TABLE numbers_${testdrive.seed} REPLICA IDENTITY FULL;
+CREATE PUBLICATION mz_source_${testdrive.seed} FOR ALL TABLES;
+INSERT INTO numbers_${testdrive.seed} VALUES (5, true, 'five');
+
+## CREATE with correct information should pass purification and fail afterwards.
+
+! CREATE MATERIALIZED SOURCE "numbers"
+  FROM POSTGRES
+    HOST 'host=${testdrive.postgres-host} user=postgres password=password dbname=postgres'
+    PUBLICATION 'mz_source_${testdrive.seed}'
+    NAMESPACE 'public'
+    TABLE 'numbers_${testdrive.seed}'
+Postgres sources not yet supported
```

Comment:
Is this and the ```CREATE MATERIALIZED SOURCE``` below valid, if you are not providing a list of columns?

Reply:
Yes! We want to allow users to elide column specification. When columns are not specified, we will pull in the information from the upstream Postgres instance via a query.